migrations/21-delete_teammates.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports.id = "delete_teammates";
+
+module.exports.up = function (done) {
+  var users = this.db.collection('users');
+  users.updateMany({}, { $unset: { 'teammates': "" } }, done);
```

Comment:
Yes this will be destroying data, but since we have database backups of all the data, if we needed to get it back I figured we could use that. The new UI won't use those fields anyway, so they would be cluttering the schema.